File: src/main/scala/io/shinto/amaterasu/enums/ActionStatus.scala
package io.shinto.amaterasu.enums

object ActionStatus extends Enumeration {
  type ActionStatus = Value
  val pending = Value("pending")
  val queued = Value("queued")
  val started = Value("started")
  val complete = Value("complete")
  val failed = Value("failed")
  val canceled = Value("canceled")
}

File: common/src/main/kotlin/org/apache/amaterasu/common/configuration/enums/ActionStatus.kt
package org.apache.amaterasu.common.configuration.enums

/**
 * Created by Eran Bartenstein on 21/10/18.
 */
enum class ActionStatus (val value: String) {
    pending("pending"),
    queued("queued"),
    started("started"),
    running("running"),
    complete("complete"),
    failed("failed"),
    canceled("canceled")
}
